src/components/editions/article/index.tsx

```diff
@@ -44,15 +44,23 @@ const articleStyles = css`
 	${articleMarginStyles}
 `;
 
-const headerStyles = css`
-	${articleWidthStyles}
-	border-bottom: 1px solid ${border.secondary};
+const hasChildNodes = (field: Option<DocumentFragment>): boolean => {
+	return field.kind === OptionKind.Some && field.value.hasChildNodes();
+};
```

Comment:
Possible abbreviation:

```suggestion
const hasChildNodes = (field: Option<DocumentFragment>): boolean =>
	field.kind === OptionKind.Some && field.value.hasChildNodes();
```